sys/mips/pmap.c

```diff
@@ -347,9 +347,10 @@ void pmap_remove(pmap_t *pmap, vaddr_t start, vaddr_t end) {
       paddr_t pa;
       if (pmap_extract_nolock(pmap, va, &pa)) {
         vm_page_t *pg = vm_page_find(pa);
-        WITH_MTX_LOCK (pv_list_lock)
+        WITH_MTX_LOCK (pv_list_lock) {
           pv_remove(pmap, va, pg);
-        pmap_pte_write(pmap, va, empty_pte(pmap), 0);
+          pmap_pte_write(pmap, va, empty_pte(pmap), 0);
```

Comment:
Do we need to hold `pv_list_lock` to call `pmap_pte_write()`?

Reply:
Hmm no. I will fix it.